share/doc/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+if(NOT VOTCA_SPHINX_DIR)
+  set(VOTCA_SPHINX_DIR "${CMAKE_CURRENT_BINARY_DIR}")
+endif()
+find_package(Python REQUIRED COMPONENTS Interpreter)
+file(GLOB XMLS ${CMAKE_CURRENT_SOURCE_DIR}/../xtp/xml/*.xml)
+set(XML_RST_FILES)
+message(status "VOTCA_SPHINX_DIR: ${VOTCA_SPHINX_DIR}")
```

Comment:
Why? Not useful for the user.

Reply:
sorry I forgot about it. I am going to delete it, thanks!